ios/FluentUI/Tooltip/TooltipViewController.swift

```diff
@@ -147,7 +130,36 @@ class TooltipView: UIView {
         set { }
     }
 
-    private func updateAppearance(tokenSet: TooltipTokenSet) {
+    func updateAppearance(tokenSet: TooltipTokenSet) {
+        // Update tokenSet
+        self.tokenSet = tokenSet
+
+        // Update tooltip size
+        positionController.updateArrowDirectionAndTooltipSize(for: message, title: titleMessage, tokenSet: tokenSet)
```

Comment:
having the position of arrow direction to the framesize itself to be in a different smaller private function makes sense to me. it feels werid why updateAppearance(token) which is meant to be more about token update dictates the frame and size.

Reply:
if message or title textstyle token changes, size also changes and if size changes too much, arrow direction may need to change.